### PR TITLE
Add GraphicsToolsMaterial Create Feature

### PR DESCRIPTION
**Pull request by HoloMoto**
## Overview

Added a new method to com.microsoft.mrtk.graphicstools.unity/Editor/Utilities/DevelopmentUtilities.cs to create MRGT materials from the Create command.

Based on the function and name, we have determined that this class is the best choice, but if there are any problems, we will add a separate class implementing the function!

## Changes

- Fixes: #170  .

## Verification

無題の動画 (1)

### CODE REVIEW

File: com.microsoft.mrtk.graphicstools.unity/Editor/Utilities/DevelopmentUtilities.cs [hunk 1]
```diff
@@ -92,7 +92,36 @@ public static void ShowSamples()
                 Debug.LogException(e);
             }
         }
+        /// <summary>
+        /// Generates a material for GraphicsToolsStandardShader in the Project window.
+        /// </summary>
+        [MenuItem("Assets/Create/Material(Graphics Tools)", false, 1)]
```
Comment (breezinstein) [inline com.microsoft.mrtk.graphicstools.unity/Editor/Utilities/DevelopmentUtilities.cs:98]: I think "Assets/Create/Graphics Tools/Material" would be a better path here

**Comment by HoloMoto** [comment]
@Cameron-Micka @breezinstein 
Corrected to reflect fb.

Sorry for being so late.
Please check it out！

**Comment by breezinstein** [comment]
> @Cameron-Micka @breezinstein Corrected to reflect fb.
> 
> Sorry for being so late. Please check it out！

Looks good to me

**Review by breezinstein — APPROVED** [review]

**Review by Cameron-Micka — APPROVED** [review]

**Comment by Cameron-Micka** [comment]
Thank you @HoloMoto!

**Pull request merged**